### PR TITLE
feat: add code formatting support

### PR DESCRIPTION
**Pull request by yeliulee**

### CODE REVIEW

File: front-end/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Function-Stream
```
Comment (RobertIndie) [inline front-end/README.md:1]:
```suggestion
# Function-Stream front end
```

File: front-end/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+## Editor Config
+
+### IDEA / WebStorm
+
+install plugin `prettier` (may have been pre-installed in webstorm)
```
Comment (RobertIndie) [inline front-end/README.md:8]: Please use the format like `* Install plugin `prettier` (may have been pre-installed in webstorm)`

**Review by RobertIndie — CHANGES_REQUESTED** [review]

**Review by RobertIndie — APPROVED** [review]
LGTM

File: front-end/.eslintrc.js [hunk 3]
```diff
@@ -3,9 +3,7 @@ module.exports = {
   env: {
     node: true
   },
-  'extends': [
-    'plugin:vue/vue3-strongly-recommended',
-  ],
+  extends: ['plugin:vue/vue3-recommended', '@vue/prettier', 'eslint:recommended'],
   rules: {
     'no-console': 'off',
```
Comment (winnd) [inline front-end/.eslintrc.js:8]: Could you please format the whole file, remove all quotes? Since this file has been modified.

**Review by winnd — APPROVED** [review]
Great, LGTM, Thanks for your contribute ;)

**Pull request merged**